test/prodigy-process-handling-test.el

```diff
@@ -34,22 +34,22 @@
 ;;;; prodigy-start-service
 
 (ert-deftest-async prodigy-start-service-test/callback-when-started (done)
-  (with-sandbox
-   (let ((service (prodigy-test/make-service)))
-     (prodigy-start-service service
-       (lambda ()
-         (prodigy-stop-service service nil done))))))
+                   (with-sandbox
```

Comment:
`ert-deftest-async` is a macro defined in some helper file.  You need to load that file before reindenting as that will also load the macro's indent rule.  This is most certainly not correct.